apps/projects/templates/a4_candy_projects/includes/project_list_tile.html

```diff
@@ -31,27 +31,30 @@
         <h3 class="tile__title" id="project-title-{{ project.pk }}">
             <a href="{{ project|project_url }}">{{ project.name }}</a>
         </h3>
-        <p class="tile__text">{{ project.description }}</p>
 
         {% if project.running_modules %}
         <div class="status-item">
-            <div class="status-bar__active">
-                <span class="status-bar__active-fill" style="width:{{ project.module_running_progress }}%"></span>
+            {% if project.latest_comments_num %}
+            <div class="d-flex justify-content-between my-1">
+                <span class="status-bar__status">
+                    <i class="far fa-comment" aria-label="Latest comments"></i>
+                    {{ project.latest_comments_num }}
+                    {% blocktrans count count=project.latest_comments_num %}
+                        new comment
+                    {% plural %}
+                        new comments
+                    {% endblocktrans %}
+                </span>
             </div>
-            <div class="d-flex justify-content-between">
-                <span class="status-bar__status"><i class="fas fa-clock" aria-hidden="true"></i>
+            {% endif %}
+            <div class="d-flex justify-content-between my-1">
+                <span class="status-bar__status"><i class="far fa-clock" aria-hidden="true"></i>
```

Comment:
when there is a longer title everything is pushed out of the tile, on design tiles are different ratio so appear taller, so we can maybe do same here?